morphia/src/test/java/org/mongodb/morphia/TestIdField.java

```diff
@@ -96,12 +96,12 @@ public void testMapAsId() throws Exception {
     @Entity
```

Comment:
I'm not clear how anything from here down is related to the index changes above.